### ISSUE TITLE
Every scraper started failing with a buildpack error

### ISSUE THREAD

**Issue by henare** [openaustralia/morph#984]
[As reported by](https://help.morph.io/t/hundreds-of-scrapers-failing-with-buildpack-error/207) @tmtmtmtm, every scraper started failing over the weekend with buildpack errors like this:

    /tmp/buildpacks/heroku-buildpack-ruby/lib/language_pack/shell_helpers.rb:49:in `run!': Command: 'set -o pipefail; curl --fail --retry 3 --retry-delay 1 --connect-timeout 30 --max-time 180 https://s3-external-1.amazonaws.com/heroku-buildpack-ruby/bundler-1.7.12.tgz -s -o - | tar zxf - ' failed unexpectedly: (LanguagePack::Fetcher::FetchError)

    gzip: stdin: unexpected end of file
    tar: Child returned status 1
    tar: Error is not recoverable: exiting now

**Comment by henare** [comment]
I said yesterday:

>On a scraper I tested I noticed an error about SSL so I decided to stop redirecting traffic to the MITM proxy that logs what domain scrapers connect to. That seems to have fixed things up for the moment.
>
>It's Sunday night here so I'll look into this more tomorrow and work on re-enabling the MITM proxy.

Today I looked into this further and discovered it's because the [MITM proxy's certificate expired](https://github.com/openaustralia/morph-docker-buildstep-base/commit/328de85860a6b642e38858dfe4fa0f7767ca9d37#diff-3254677a7917c6c01f55212f86c57fbfR12) :cry: 

So we need to update that certificate, push it to that repo, that will trigger a rebuild of the Docker image, then we need to pull that onto the morph.io server.

**Comment by henare** [comment]
Last night I:

* Updated the container certificate: https://github.com/openaustralia/morph-docker-buildstep-base/commit/e226708d8f9dd32261da113193685165cd335f23
* Updated the mitmproxy certificates: https://github.com/openaustralia/morph/commit/3b590e1beb54e91ef8119b4a0f5d2fc418a90568
* Deployed (which also pulls down updated docker image)
* Re-enabled mitmproxy `iptables-morph-add`
* Tested a broken scraper - it was working again
* Tried a scraper in another language - it had a different error: https://morph.io/morph-test-scrapers/test-python

```
 Traceback (most recent call last):
  File "scraper.py", line 1, in <module>
    import scraperwiki
ImportError: No module named scraperwiki
```

* So I disabled the mitmproxy again so I could start debugging again today

And wrote some readme on how to update the certificate again: https://github.com/openaustralia/morph-docker-buildstep-base/commit/2fb52b74b612a96a74019eb52da7fb9092c46760

**Comment by henare** [comment]
>Tried a scraper in another language - it had a different error: https://morph.io/morph-test-scrapers/test-python

I think this might be a separate issue: https://help.morph.io/t/failed-module-import/206

I've confirmed it works locally but not on production. That makes me think it's a problem with the cached Docker image.

**Comment by henare** [comment]
>I think this might be a separate issue: https://help.morph.io/t/failed-module-import/206

I get the same issue with the mitmproxy enabled or disabled. I'm saying this issue is closed and opening #985 to track that issue.

**Comment by henare** [comment]
Eugh, in trying to fix #985 I just tried to run [the PHP test scraper](https://morph.io/morph-test-scrapers/test-php). It wasn't working because the buildpack wouldn't install with mitmproxy enabled.

Disabling the mitmproxy fixed the scraper and now it will run with or without the mitmproxy enabled, presumably because the image is cached.
